frontend/main-app/src/components/DashboardAccordion/DashboardAccordion.tsx

```diff
@@ -13,20 +13,21 @@ import axios from "axios";
 import { CustomParagraph } from "../CustomParagraph";
 import { SubmissionViewer } from "../SubmissionViewer";
 import { ShareForm } from "./ShareForm";
+import { DeleteForm } from "../DeleteForm/DeleteForm";
+import { FormContext } from "../Dashboard";
+import { t } from "i18next";
 interface ButtonGroupProps {
   submissions: Submission[];
-  formTitle: string;
   formUrl: string;
 }
 
-const ButtonGroup: React.FC<ButtonGroupProps> = ({ submissions, formTitle, formUrl }) => {
+const ButtonGroup: React.FC<ButtonGroupProps> = ({ submissions, formUrl }) => {
```

Comment:
I think we can refactur `DashboardAccordion `by moving `ButtonGroup `to a seperate component. I see that you did not build `ButtonGroup `in this PR , so maybe we can add a seperate  quilty code issue  and fix it later. 😊

Reply:
Agreed, I did that in PR #164 😊